phase-6/transfer-dtag.md

```diff
@@ -0,0 +1,157 @@
+
+1. A profile with a DTag properly set. 
+2. A friend with which to share your DTag that will need to accept the request **or** a second profile with a DTag set.
+  
+In the following commands examples we will use two local keys, each one with a profile set to it. If you want to do the same, in order to register a new key use the following command: 
+
+```shell
+desmoscli keys add <new-key-name>
+```  
+
+:::
+
+
+### Transfer your DTag
+Transferring a DTag is a process composed of two different steps: 
+
+1. Preparing for the transfer by executing a `transfer-dtag` transaction. 
+2. Accepting the transferring by performing an `accept-dtag-transfer` transaction. 
+
+So, firstly we need to prepare for the DTag transferring. You can do so by executing the following command:  
+
+```bash
+desmoscli tx profiles transfer-dtag <User address> \
+  --from <your-key-name> --yes \
+  --chain-id <chain-id>
+```
+
+The `<User address>` should be the address of the DTag recipient.
+
+Here's an example of such command: 
+
+```bash
+desmoscli tx profiles transfer-dtag $(desmoscli keys show alice -a) \
+  --from jack --yes \
+  --chain-id morpheus-10000
+```
+
+In this case, we are transferring the DTag from Jack to Alice.
+
+Once you've run that command you will be asked to type the password you've chosen during the setup and after having inserted it properly you should see something like this: 
+
+```yml
+height: 0
+txhash: E4920B61BFCFB935124FFC22BC04C1D5619943B138C87913E6EBCF2B5E892290
+codespace: ""
+code: 0
+data: ""
+rawlog: '[]'
+logs: []
+info: ""
+gaswanted: 0
+gasused: 0
+tx: null
+timestamp: ""
+```
+
+To make sure the transaction has been processed successfully, you can query it using the following command: 
+
+```bash
+desmoscli query tx <hash> --trust-node --output json
+
+# Example
+# desmoscli query tx E4920B61BFCFB935124FFC22BC04C1D5619943B138C87913E6EBCF2B5E892290 \
+  --trust-node --output json
+``` 
+
+This will return you the JSON representation of the transaction itself.
+
+### Accepting the transferring
+Once that you started the DTag transferring, the receiving user needs to accept it. To do so, they can use the following command:
+
+```bash
+desmoscli tx profiles accept-dtag-transfer.md <New DTag> <Sender address> \
+  --from <their-key-name> --yes \
+  --chain-id <chain-id>
+```
+
+:::warning Note  
+As you might see, the command ends with `.md`. This is **not** an error in the documentation. This is a bug that is currently present inside `desmoscli`.  
+In the future that `.md` suffix will be removed, but for now you need to use it unless specified differently.  
+:::
+
+The `<New DTag>` will be the new DTag that is going to be associated to them once they gave you their current DTag.  
+The `<Sender address>` must be the same address of the user that has started the DTag transferring. 
+
+Here's an example of such command: 
+
+```shell
+desmoscli tx profiles accept-dtag-transfer.md \
```

Comment:
Would this https://github.com/desmos-labs/desmos/pull/282 be tagged and refactor the correct command here?

Reply:
Yeah I think it's better to tag v0.12.4 and tell people to use that when running this challenges

Reply:
I've added to the Phase 6 README file that all the challenges must be done using Desmos v0.12.3 which we will tag shortly